tier2/{{cookiecutter.project_slug}}/CONTRIBUTING.md

```diff
@@ -12,35 +15,46 @@ We encourage you to read this project's CONTRIBUTING policy (you are here), its
 -->
 
 ## Getting Started
-<!--- ### TODO: If you have 'good-first-issue' or 'easy' labels for newcomers, mention them here.-->
 
+<!--- TODO: If you have 'good-first-issue' or 'easy' labels for newcomers, mention them here.-->
+
+<!-- 
 ### Team Specific Guidelines
-<!--- ### TODO -->
+
+TODO: This section helps contributors understand any team structure in the project (formal or informal.) Encouraged to point towards the MAINTAINERS.md file for further details. 
+-->
 
 ### Building dependencies
-<!--- ### TODO -->
+
+<!--- TODO: This step is often skipped, so don't forget to include the steps needed to install on your platform. If you project can be multi-platform, this is an excellent place for first time contributors to send patches!-->
 
 ### Building the Project
-<!--- ### TODO -->
+
+<!--- TODO: Be sure to include build scripts and instructions, not just the source code itself! -->
 
 ### Workflow and Branching
+
 <!--- TODO: Workflow Example
 We follow the [GitHub Flow Workflow](https://guides.github.com/introduction/flow/)
 
 1.  Fork the project 
-1.  Check out the `main` branch 
-1.  Create a feature branch
-1.  Write code and tests for your change 
-1.  From your branch, make a pull request against `cmsgov/cmsgov-example-repo/main` 
-1.  Work with repo maintainers to get your change reviewed 
-1.  Wait for your change to be pulled into `cmsgov/cmsgov-example-repo/main`
-1.  Delete your feature branch
+2.  Check out the `main` branch 
```

Comment:
using `1.` will (should?) auto-number elements in a markdown list, but I'm ok with explicit numbering too :)